### PR TITLE
Pressure Stall Information and Conntrack Information added to vm-status.json, relabeled some rows to reflect content

### PR DESCRIPTION
**Pull request by elikoga**
PL-129904 Replace check_load with check_psi

PL-130189 stateful connections too early in firewall rules?

### CODE REVIEW

File: vm-status.json [hunk 1]
```diff
@@ -2010,7 +2789,7 @@
     ]
   },
   "time": {
-    "from": "now-1h",
+    "from": "now-2d",
```
Comment (zagy) [inline vm-status.json]: This looks not intentional

File: vm-status.json [hunk 2]
```diff
@@ -2039,6 +2818,7 @@
     ]
   },
   "timezone": "browser",
-  "title": "VM Status",
-  "version": 52
-}
+  "title": "VM Status With Pressure Data (ekw) WIP",
```
Comment (zagy) [inline vm-status.json]: Probably not.

File: vm-status.json [hunk 3]
```diff
@@ -31,1975 +30,2755 @@
...
     "list": [
       {
         "allValue": null,
         "current": {
-          "tags": [],
-          "text": "",
-          "value": ""
+          "selected": false,
+          "text": "test28",
+          "value": "test28"
```
Comment (zagy) [inline vm-status.json]: should be empty

**Comment by elikoga** [comment]
looks more ready now, adjusted id to upstream id

**Review by zagy — APPROVED** [review]

**Pull request merged**